src/main/java/it/pn/frontend/e2e/pages/mittente/PiattaformaNotifichePage.java

```diff
@@ -557,6 +557,7 @@ public void siCambiaIlNumeroElementiVisualizzatiAttraversoIlFiltro() {
         this.getWebDriverWait(10)
                 .withMessage("Il pulsante 'righe per pagina' non è presente")
                 .until(ExpectedConditions.visibilityOf(this.numeroNotificheButton));
+        this.js().executeScript("arguments[0].scrollIntoView(true);", this.numeroNotificheButton);
```

Comment:
in this part, add vaiInFondoAllaPagina() (you find him thanks to BasePage inheritance) method and then click button